indicators/NO_GLAC_001/R/references.bib

```diff
@@ -0,0 +1,90 @@
+	date = {2008},
+	journal = {Journal of Biogeography},
+	pages = {1906--1922},
+	volume = {35},
+	number = {10},
+	doi = {10.1111/j.1365-2699.2008.01941.x},
+	url = {https://onlinelibrary.wiley.com/doi/abs/10.1111/j.1365-2699.2008.01941.x},
+	note = {{\_}eprint: https://onlinelibrary.wiley.com/doi/pdf/10.1111/j.1365-2699.2008.01941.x},
+	langid = {en}
+}
+
+@book{nybø2017,
+	title = {Fagsystem for fastsetting av god økologisk tilstand - forslag fra et ekspertråd},
+	author = {{Nybø}, Signe and {Evju (red)}, Marianne},
+	year = {2017},
+	date = {2017},
+	publisher = {Ekspertrådet for Økologisk Tilstand},
+	url = {https://www.regjeringen.no/no/dokumenter/fagsystem-for-fastsetting-av-god-okologisk-tilstand/id2558481/}
+}
+
+@article{zemp2025,
+	title = {Community estimate of global glacier mass changes from 2000 to 2023},
+	author = {Zemp, Michael and Jakob, Livia and Dussaillant, {Inés} and Nussbaumer, Samuel U. and Gourmelen, Noel and Dubber, Sophie and A, Geruo and Abdullahi, Sahra and Andreassen, Liss Marie and Berthier, Etienne and Bhattacharya, Atanu and Blazquez, Alejandro and Boehm Vock, Laura F. and Bolch, Tobias and Box, Jason and Braun, Matthias H. and Brun, Fanny and Cicero, Eric and Colgan, William and Eckert, Nicolas and Farinotti, Daniel and Florentine, Caitlyn and Floricioiu, Dana and Gardner, Alex and Harig, Christopher and Hassan, Javed and Hugonnet, Romain and Huss, Matthias and {Jóhannesson}, {Tómas} and Liang, Chia-Chun Angela and Ke, Chang-Qing and Khan, Shfaqat Abbas and King, Owen and Kneib, Marin and Krieger, Lukas and Maussion, Fabien and Mattea, Enrico and McNabb, Robert and Menounos, Brian and Miles, Evan and Moholdt, Geir and Nilsson, Johan and {Pálsson}, Finnur and Pfeffer, Julia and Piermattei, Livia and Plummer, Stephen and Richter, Andreas and Sasgen, Ingo and Schuster, Lilian and Seehaus, Thorsten and Shen, Xiaoyi and Sommer, Christian and Sutterley, Tyler and Treichler, {Désirée} and Velicogna, Isabella and Wouters, Bert and Zekollari, Harry and Zheng, Whyjay and The GlaMBIE Team, },
+	year = {2025},
+	month = {03},
+	date = {2025-03},
+	journal = {Nature},
+	pages = {382--388},
+	volume = {639},
+	number = {8054},
+	doi = {10.1038/s41586-024-08545-z},
+	url = {https://www.nature.com/articles/s41586-024-08545-z},
+	note = {Publisher: Nature Publishing Group},
+	langid = {en}
+}
+
+@article{framstad2021,
+	title = {Vurdering av økologisk tilstand for skog i Norge i 2020},
+	author = {Framstad, Erik and Berglund, {Håkan} and Jacobsen, Rannveig M and Jakobsson, Simon and Ohlson, Mikael and Sverdrup-Thygeson, Anne and {Töpper}, Joachim},
+	year = {2021},
+	date = {2021},
+	series = {NINA Rapport},
+	volume = {2000},
+	note = {Publisher: Norsk institutt for naturforskning (NINA)}
+}
+
+@article{openshaw1984,
+	title = {The modifiable areal unit problem},
+	author = {Openshaw, Stan},
+	year = {1984},
+	date = {1984},
+	journal = {Concepts and techniques in modern geography},
+	note = {Publisher: GeoBooks}
+}
```

Comment:
Add 
@book{macarthur2001theory,
  title={The theory of island biogeography},
  author={MacArthur, Robert H and Wilson, Edward O},
  volume={1},
  year={2001},
  publi

Reply:
Since I changed the reasoning for the x60 value I actually dont need this ref in the end.